crates/bevy_transform/src/components/transform.rs

```diff
@@ -117,16 +117,24 @@ impl Transform {
         }
     }
 
-    /// Updates and returns this [`Transform`] by rotating it so that its unit
-    /// vector in the local negative `Z` direction is toward `target` and its
-    /// unit vector in the local `Y` direction is toward `up`.
+    /// Returns this [`Transform`] with a new rotation so that [`Transform::forward`]
+    /// points towards the `target` position and [`Transform::up`] points towards `up`.
```

Comment:
IMO we should also keep the local z etc explanations, probably in parentheticals.

Reply:
This is already documented on the linked methods. I'd rather avoid repeating the documentation.
Although, since rust-analyzer doesn't support these links it does kinda suck. hmmmm